pkg/deployment/openstack.go

```diff
@@ -140,6 +140,7 @@ func RunOpenStack(ctx context.Context, helper *helper.Helper, obj client.Object,
 		Strategy:       "linear",
 		GatherFacts:    false,
 		AnyErrorsFatal: true,
+		Become:         true,
```

Comment:
Do you envision that we will do this temporarily until we have ironed out the privilege escalation stuff in edpm_ansible?

Reply:
Yes and also these would be moving to dataplaneservice interface/CRs soon. This is a WA for the time being to unblock.

Reply:
We could have removed https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/pkg/deployment/os.go#L203, but that breaks a few roles. So the plan is to fix roles not to require become:true at the play level.

Reply:
https://github.com/openstack-k8s-operators/edpm-ansible/pull/158 would be probably enough to remove Become:true from RunOS tasks. We'll see.

Reply:
Looks like more work required with some other roles which I'll work next week. We should merge this IMO.